app/services/search/vacancy_search.rb

```diff
@@ -67,8 +67,8 @@ def search_strategy
                              Search::Strategies::PgSearch.new(
                                keyword,
                                filters: search_criteria,
-                               location: search_criteria[:location],
-                               radius: search_criteria[:radius],
+                               location: location_search.location_filter[:location],
+                               radius: location_search.radius,
```

Comment:
@csutter `location_search` has some logic in it (from `Search::LocationBuilder`) that alters the radius attribute. Should we
(a) pass in this attribute to `PgSearch`, as above, or
(b) replicate this logic within `VacancyLocationQuery`?

Reply:
I reckon a) does the job for now.

My general approach for the `PgSearch` stuff has been to reuse as much as we can from the existing logic, and then when we go all in on the new search, slowly unpicking everything into new and possibly nicer code!